client/hotrod-client/src/main/java/org/infinispan/client/hotrod/RemoteCacheManager.java

```diff
@@ -103,11 +103,14 @@ public Configuration getConfiguration() {
       return configuration;
    }
 
-
    /**
+    * Retrieves a clone of the properties currently in use.  Note that making
+    * any changes to the properties instance retrieved will not affect an
+    * already-running RemoteCacheManager.
+    *
+    * @return a clone of the properties used to configure this RemoteCacheManager
     * @since 4.2
     */
-   @Override
    @Deprecated
    public Properties getProperties() {
```

Comment:
Removing this method from RemoteCacheManager can be done independently of this PR, which is only concerned with the RemoteCacheContainer interface.